clusterInfo.py

```diff
@@ -20,8 +23,12 @@ def __init__(self, name: str):
         self.bmcs = []  # type: list[str]
 
 
-def _default_cred_paths() -> list[str]:
+def _default_cred_paths(*, honor_env: bool = True) -> list[str]:
     paths = []
+    if honor_env:
+        p = os.environ.get(ENV_CDA_CLUSTERINFO_CREDENTIALS, None)
```

Comment:
Where is this used?

Reply:
it's not used per-se. It's more for an end-user (or a end-user's script) to configure another credentials file.

For example, `CDA_CLUSTERINFO_CREDENTIALS=/tmp/cred.json make e2e-tests` (in dpu-operator) would work, where this setting is otherwise not configurable.

Reply:
Let's move that to a separate pr where we introduce a cda level config